client/src/ui/components/bank/ResourceBar.tsx

```
@@ -5,152 +5,154 @@
-          max={max}
-          arrows={false}
-          allowDecimals
-          onFocus={onFocus}
-          onBlur={onBlur}
-        />
+          {!disableInput && (
+            <div
+              className="flex text-xs text-gold/70 mt-1 justify-center items-center relative text-center self-center mx-auto w-full cursor-pointer"
+              onClick={() => handleAmountChange(finalResourceBalance)}
+            >
+              Max: {isNaN(selectedResourceBalance) ? "0" : selectedResourceBalance.toLocaleString()}
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)